_posts/2015-03-06-credits.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Credits"
+bg: blue  #defined in _config.yml, can use html color like '#0fbfcf'
+color: white   #text color
+fa-icon: wrench
+---
+
+The contents of this landing page are derived from the "[Women In Tech](http://witchat.github.io/)" Slack group landing page.
```

Comment:
Perfect - can you also credit Rachel here by her twitter handle? https://twitter.com/zenparty I checked in with her yesterday and she's happy we are using it, and said that was a good way to give her credit :)